packages/protocol/contracts/layer1/core/impl/Inbox.sol

```diff
@@ -263,26 +272,15 @@ contract Inbox is IInbox, IForcedInclusionStore, EssentialContract {
             );
 
             // ---------------------------------------------------------
-            // 3. Calculate proposal age and bond instruction
+            // 3. Calculate proposal age and process bond instruction
             // ---------------------------------------------------------
-            Transition memory transitionAtOffset = commitment.transitions[offset];
-            uint256 proposalAge =
-                block.timestamp - transitionAtOffset.timestamp.max(state.lastFinalizedTimestamp);
 
-            // Bond transfers only apply to the first newly-finalized proposal.
-            LibBonds.BondInstruction memory bondInstruction =
-                LibBondInstruction.calculateBondInstruction(
-                    commitment.firstProposalId + offset,
-                    proposalAge,
-                    transitionAtOffset.proposer,
-                    transitionAtOffset.designatedProver,
-                    commitment.actualProver,
-                    _provingWindow,
-                    _extendedProvingWindow
-                );
-            if (bondInstruction.bondType != LibBonds.BondType.NONE) {
-                _signalService.sendSignal(LibBonds.hashBondInstruction(bondInstruction));
-                emit BondInstructionCreated(bondInstruction.proposalId, bondInstruction);
+            // `proposalAge` is the age of the next-to-finalize proposal.
+            uint256 proposalAge = block.timestamp
+                - commitment.transitions[offset].timestamp.max(state.lastFinalizedTimestamp);
```

Comment:
@ggonzalez94 maybe we should simply use: `block.timestamp - commitment.transitions[offset].timestamp`?

Anyway, this may related to the PR you will rebase.